charts/mattermost-rtcd/values.yaml

```diff
@@ -44,8 +44,11 @@ configuration:
   environmentVariables: {}
     # All env vars that exist for RTCD: https://github.com/mattermost/rtcd/blob/master/docs/env_config.md
     # RTCD_API_SECURITY_ALLOWSELFREGISTRATION: "\"true\""
+    # RTCD_RTC_ICESERVERS: "'[{\"urls\":[\"stun:stun.global.calls.mattermost.com:3478\"]}]'"
     # RTCD_LOGGER_CONSOLELEVEL: "\"DEBUG\""
     # RTCD_LOGGER_ENABLEFILE: "\"false\""
+    # RTCD_RTC_ICEPORTUDP: "\"8443\""
+    # RTCD_RTC_ICEPORTTCP: "\"8443\""
```

Comment:
Adding these because no matter what you configure as `RTCport` below these are the actual ports the daemon will listen on and should be updated accordingly if needed.